PowerShellTools/Classification/PowerShellParseUtilities.cs

```diff
@@ -187,6 +188,22 @@ internal static IList<ScriptParameterViewModel> GenerateCommonParameters()
                     new ScriptParameter("WarningVariable", DataTypeConstants.StringType, null, new HashSet<object>())
                     )
             };
+
+            if(DependencyUtilities.GetInstalledPowerShellVersion().Major >= 5)
+            {
+                // InformationAction
+                commonParameters.Insert(3, new ScriptParameterViewModel(

```

Comment:
As of now, the initial insertions (above) are done in sorted order and afaik, the resulting list is never sorted later. I can go in and add in some sorting logic though, shouldn't be too hard.